imap_processing/utils.py

```diff
@@ -254,3 +255,86 @@ def update_epoch_to_datetime(dataset: xr.Dataset) -> xr.Dataset:
     dataset = dataset.assign_coords(epoch=epoch)
 
     return dataset
+
+
+def packet_file_to_datasets(
```

Comment:
It may be useful to optionally skip an apid. The motivation for this is packets with variable length binary data fields.

Reply:
This handles variable length binary data!

I can see the argument here as well, but again I think that it might be easier for a user to just not access that apid in the dictionary lookup?